templates/html_report/index.php

```diff
@@ -159,68 +180,32 @@
-                                <?php if (0 !== count($packagesInstalled)) {?><th>Installed</th><?php } ?>
-                                <th>Latest</th>
-                                <th>License</th>
-                            </tr>
-                            </thead>
-                            <tbody id="contentPackages" class="clusterize-content">
-                            <?php
-                            $packages = isset($project['composer']['packages']) ? $project['composer']['packages'] : [];
-                            usort($packages, function ($a, $b) {
-                                return strcmp($a->name, $b->name);
-                            });
-                            foreach ($packages as $package) if( !preg_match('!(^php$|^ext\-)!', $package->name)) { ?>
-                                <tr>
-                                    <td>
-                                        <a target="_blank" href="https://packagist.org/packages/<?php echo $package->name; ?>">
-                                            <?php echo $package->name; ?>
-                                        </a>
-                                    </td>
-                                    <td><?php echo $package->required; ?></td>
-                                    <?php if (0 !== count($packagesInstalled)) {?><td><?php echo $package->installed; ?></td><?php } ?>
-                                    <td><?php echo $package->latest; ?></td>
-                                    <td><?php foreach($package->license as $license) { ?>
-                                            <a target="_blank" href="https://spdx.org/licenses/<?php echo $license;?>.html"><?php echo $license;?></a>
-                                        <?php } ?>
-                                    </td>
-                                </tr>
-                            <?php } ?>
-                            </tbody>
-                        </table>
-                        <?php if(0 === count($packages)) { ?>
-                            <div>No composer.json file found</div>
-                        <?php } ?>
-                        <?php if(0 === count($packagesInstalled)) { ?>
-                            <div>No composer.lock file found</div>
-                        <?php } ?>
+        <?php if($this->isHomePage()) {?>
+            <div class="column">
+                <div class="bloc bloc-number">
+                    <div class="label">
+                        <a href="composer.html">Composer</a>
                     </div>
-                </div>
-            </div>
-        </div>
+                    <?php
+                    $packages = isset($project['composer']['packages']) ? $project['composer']['packages'] : [];
+                    $packagesInstalled = isset($project['composer']['packages-installed']) ? $project['composer']['packages-installed'] : [];
+                    if([] === $packages) { ?>
```

Comment:
```suggestion
                    if ([] === $packages) { ?>
```
But this probably comes from the #424. If there is a dependency with a rebase required, please let it go.